app/api/v2/entities/organization.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module API
+  module V2
+    module Entities
+      class Organization < API::V2::Entities::Base
+        expose :id,
+               documentation: {
+                 type: 'Integer',
+                 desc: 'Organization ID'
+               }
+
+        expose :oid,
+               documentation: {
+                 type: 'String',
+                 desc: 'Organization OID'
```

Comment:
do we need more params here?
like address,city...

Reply:
I do as base class for Organization. It could be expose more of detials in `app/api/v2/commercial/entities`